test/jdk_trusted.cnf

```diff
@@ -0,0 +1,8 @@
+#
```

Comment:
Please create a new sub-directory test/recipes/80-test_pkcs12_data, and move this file to there.